src/tokens.ts

```diff
@@ -176,7 +176,7 @@ export function tokensFor(d: Dialect) {
     let {next} = input
     input.advance()
     if (inString(next, Space)) {
-      while (inString(input.next, Space)) input.advance()
+      input.acceptToken(Whitespace)
```

Comment:
The original [keeps moving the pointer forward as long as the next char is any of ` \t\r\n`](https://github.com/codemirror/lang-sql/blob/be533e3e3bd220e57de0b82300f6493308aac6e7/src/tokens.ts#L179), which I expect creates a single whitespace node out of multiple space chars in a row, instead of one for each.

Should we preserve that behavior?

Reply:
No reason not to. Adding it back.